### ISSUE TITLE
"Tamper protection" feature

### ISSUE THREAD

**Issue by richardwilly98** [richardwilly98/es-dms#26]
Prevent unauthorized access to data stored in repository

Create a new document attribute computing hash of the json document.

**Comment by danilos** [comment]
good idea, what about using encryption and decryption at client side independent of SSL?
